pkg/quota/util/helper.go

```diff
@@ -0,0 +1,30 @@
+package util
+
+import (
+	"strings"
+
+	apierrs "k8s.io/kubernetes/pkg/api/errors"
+)
+
+// errMessageString is a part of error message copied from quotaAdmission.Admit() method in
+// k8s.io/kubernetes/plugin/pkg/admission/resourcequota/admission.go module
+const errQuotaMessageString = `exceeded quota:`
+const errQuotaUnknownMessageString = `status unknown for quota:`
+const errLimitsMessageString = `exceeds the maximum limit`
+
+// IsErrorQuotaExceeded returns true if the given error stands for a denied request caused by detected quota
+// abuse.
+func IsErrorQuotaExceeded(err error) bool {
+	if isForbidden := apierrs.IsForbidden(err); isForbidden || apierrs.IsInvalid(err) {
```

Comment:
I'm aware of that. That's why there's a [unit test](https://github.com/openshift/origin/pull/8195/files/e0ee942b49f3852ba860c379c119fbf20df77998..9ad4b0157a8aceb5bb632ec6f9ccc38d912c563b#diff-fd1dfc5323b0c2eb9545baa069dcdcf8R16) and [another](https://github.com/openshift/origin/pull/8195/files#diff-199bfaa771b0dc11f73e8d1369e5aae8R245) and [yet another](https://github.com/openshift/origin/pull/8195/files#diff-54647b717d5e3f5518b97fd188d8743bR19) and [an extended test](https://github.com/openshift/origin/pull/8195/files#diff-5cd84bb51adaca2feca29e9a60911eabR165) and [integration test](https://github.com/openshift/origin/pull/8195/files#diff-6df5bcb947acc5e4eda3319f71d9a9abR114) as well.